modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-dialect/src/main/resources/com/liferay/fragment/collection/contributor/dialect/dependencies/dialect-tabs/index.html

```diff
@@ -0,0 +1,38 @@
+<div class="component-tabs ${(configuration.tabDirection == 'nav-vertical')?then('d-flex', '')}">
+	<nav class="navbar navbar-collapse-absolute navbar-expand-md p-0">
+	<button aria-controls="navigationBarCollapse-${fragmentEntryLinkNamespace}" aria-expanded="false" aria-label="Tabs Toggle Navigation" class="collapsed navbar-toggler navbar-toggler-link nav nav-${configuration.tabStyle} ${configuration.tabDirection} ${(configuration.tabActiveBarPosition)}  p-0" data-toggle="collapse">
+			<span class="nav-link active navbar-text-truncate btn btn-unstyled">Tab 1</span>
```

Comment:
sort classes whenever possible

Reply:
Also this looks like it was changed from the tab fragment in portal. What are the reasons for this change? It seems odd to have btn classes on a span inside of a button element.

Reply:
Fixed